src/RpcClient/Models/RpcVersion.cs

```diff
@@ -14,27 +44,28 @@ public class RpcVersion
 
         public string UserAgent { get; set; }
 
+        public RpcProtocol Protocol { get; set; } = new();
 
         public JObject ToJson()
         {
             JObject json = new();
-            json["network"] = Network;
```

Comment:
This will break compatibility, do we really want that?

Reply:
We can create `getprotocol`

Reply:
I'd rather just return `network` here and in `protocol` subsection, make a notice that top-level `network` is deprecated and will be removed and then remove it in some subsequent release.